### PR TITLE
added text showing time since last donation on dashboard

### PR DESCRIPTION
**Pull request by ErikBjare**
Also fixes bug with overly eager reminders

### CODE REVIEW

File: src/lib/util.ts [hunk 1]
```diff
@@ -35,3 +37,13 @@ export async function getInstallDate(): Promise<Date> {
...
 }
+
+export async function lastDonationDate(db): Promise<Date | null> {
+  let donations = await db.getDonations(1);
+  return donations.length > 0 ? donations[0].date : null;
+}
+
+export async function secondsSinceDonation(db): Promise<number> {
+  const lastTime = (await lastDonationDate(db)) || (await getInstallDate());
+  return moment().diff(new Date(lastTime)) / 1000;
```
Comment (dwilid) [inline src/lib/util.ts]: Maybe just keep it in milliseconds and rename function since the only place it's used at the moment converts it back to millis

Comment (patrik-laurell) [reply src/lib/util.ts]: in mounted funciton in dashboard page it is used to get seconds. could easily be changed to millis there as well though

File: src/dashboard/pages/DashboardPage.vue [hunk 2]
```diff
@@ -47,13 +47,15 @@ div
     v-container(grid-list-md)
       // Favorite creators
       v-layout(row)
-        v-flex(xs12).mb-6
-          div.display-1 Your favorite creators
-        v-flex(grow)
+        v-flex
+          span.display-1 Your favorite creators
+          span.ml-3.caption(v-show="daysSinceDonation !== null")
+            | You last donated {{daysSinceDonation}} days ago
```
Comment (patrik-laurell) [inline src/dashboard/pages/DashboardPage.vue]: Maybe find different placement for the text? Inside donation summary?

Comment (ErikBjare) [reply src/dashboard/pages/DashboardPage.vue]: I want it to be visible from the moment you open the dashboard. DonationSummary shouldn't do even more things, if anything it should do less.

**Pull request merged**